hubble-relay.Dockerfile

```diff
@@ -1,6 +1,9 @@
+#
```

Comment:
I have to ask, why the `#` on top of all files?

Reply:
It makes it a bit easier to add the required stanza to the first line without risk of doing it if the Dockerfile already contains it.